lib/communityEvent/index.spec.js

```diff
@@ -0,0 +1 @@
+import '.'; // It doesn't crash.
```

Comment:
There's not really any mechanism for testing that stuff in badger brain yet. I could check the schema has that key but I don't think it's useful. I really wanna test the resolvers and the relationships between them in an integration sense, but it's a big job getting BB ready for that.